Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient.h

```diff
@@ -444,37 +446,20 @@ typedef void (^BUYDataGiftCardBlock)(BUYGiftCard *giftCard, NSError *error);
 - (NSURLSessionDataTask *)updateCheckout:(BUYCheckout *)checkout completion:(BUYDataCheckoutBlock)block;
 
 /**
- *  Finalizes the BUYCheckout and charges the credit card.
+ *  Finalizes the BUYCheckout and charges the payment provider (ex: Credi Card, Apple Pay, etc).
  *  This enqueues a completion job on Shopify and returns immediately.
  *  You must get the job's status by calling checkCompletionStatusOfCheckout:block
  *
  *  Note: There's no guarantee that the BUYCheckout returned will be the same as the one that is passed in.
  *  We recommended using the BUYCheckout returned in the block.
  *
- *  @param checkout The BUYCheckout to complete
- *  @param block    (^BUYDataCheckoutBlock)(BUYCheckout *checkout, NSError *error);
- *
- *  @return The associated NSURLSessionDataTask
- */
-- (NSURLSessionDataTask *)completeCheckout:(BUYCheckout *)checkout completion:(BUYDataCheckoutBlock)block;
-
-/**
- */
-/**
- *  Finalizes the checkout and charges the credit card associated with the payment token from PassKit (Apple Pay).
- *  This only enqueues a completion job, and will return immediately.
- *  You must get the job's status by calling checkCompletionStatusOfCheckout:block
- *
- *  Note: There's no guarantee that the BUYCheckout returned will be the same as the one that is passed in.
- *  We recommended using the BUYCheckout returned in the block.
- *
- *  @param checkout The BUYCheckout to complete
- *  @param token    The PKPaymentToken
- *  @param block    (^BUYDataCheckoutBlock)(BUYCheckout *checkout, NSError *error);
+ *  @param checkout      The BUYCheckout to complete
+ *  @param paymentToken  Opaque payment token object
```

Comment:
That parameters isn't for `BUYDataCheckoutBlock`. Its for the `completeCheckout:paymentToken:completion:` method below.